docs/design/coreclr/botr/clr-abi.md

```diff
@@ -585,6 +585,11 @@ The native EH clauses would be listed as follows:
 
 If the handlers were in a different order, then clause 6 might appear before clauses 4 and 5, but never in between.
 
+## Clauses covering the same try region
+
+Several consecutive clauses may cover the same `try` block. A clause covering the same region as the previous one is marked by the `COR_ILEXCEPTION_CLAUSE_SAMETRY` flag. During collided unwind when an exception is thrown while running handler for another exception, this enables the runtime to skip clauses that that cover the same `try` block as the clause that handled the original exception.
```

Comment:
```suggestion
Several consecutive clauses may cover the same `try` block. A clause covering the same region as the previous one is marked by the `COR_ILEXCEPTION_CLAUSE_SAMETRY` flag. During collided unwind when an exception is thrown while running handler for another exception, this enables the runtime to skip clauses that cover the same `try` block as the clause that handled the original exception.
```

This isn't actually a collided unwind, is it? When running the handler, the unwind has completed. It's just a nested exception.

Reply:
The NativeAOT uses the term "collided" for the case when an exception is thrown while a catch / exceptionally called finally funclet frame is found on the stack when running first pass of exception handling searching for handler of another exception and the stack walker walks past it. 
https://github.com/dotnet/runtime/blob/d389ab955dbb65547643fa1aed52669b02c04294/src/coreclr/nativeaot/Runtime/StackFrameIterator.cpp#L1374-L1393
The flag is used only in that specific case, not for nested exceptions that are fully handled in the call chain of the previous exception handler and don't escape it.  I can see that the text above isn't clear about that, let me update it.

Reply:
@BruceForstall I've updated the doc, is it clearer now?